runtime/lua/vim/uri.lua

```diff
@@ -52,7 +52,7 @@ end
 
 --@private
 local function is_windows_file_uri(uri)
-  return uri:match('^file:///[a-zA-Z]:') ~= nil
+  return uri:match('^file:/+[a-zA-Z]:') ~= nil
```

Comment:
Hmm would it be better to optionally match on [//] ?

Reply:
If I understand your suggestion correctly, that's actually what I wanted to do at first. Unfortunately Lua's pattern matching system is much less powerful than regular expressions and it does not allow optional matches (besides repetition).

Reply:
Yes, we could vim's built-in regex engine, but yes, this is probably fine. I just think it's a little weird we could match against  `file://////////`